frontend/components/Header.js

```diff
@@ -130,6 +134,9 @@ class Header extends Component {
                 <li>
                   <a href=".">Medical recovery centre</a>
                 </li>
+                <li>
+                  <a href="/resources">Resources</a>
```

Comment:
Please use next link for router rather then native browser routing because you won't get the benefits of fast routing provided by next. https://nextjs.org/docs#with-link

Reply:
Better?